packages/bvaughn-architecture-demo/src/suspense/ObjectPreviews.ts

```diff
@@ -222,7 +222,7 @@ export async function getObjectPropertyHelper(
   }
 }
 
-export function preCacheObjects(pauseId: PauseId, objects: Object[]): void {
+export function preCacheObjects(pauseId: PauseId, objects: Object[] = []): void {
```

Comment:
I don't think we should make this change.

Not that it would cause a bug, but it seems to hint at a conceptual problem. (What does it mean for code to be calling `preCacheObjects` without even passing a parameter for the objects that are being cached?)

Reply:
Yeah, I made this change because the standard `PauseData` type contains `{objects?: []}`, and I was getting annoyed at having to provide a default every time I called `preCacheObjects()`.  I can live with `preCacheObjects(pauseData.objects ?? [])`, I guess.

Reply:
Wouldn't the more semantic thing to do be
```js
if (pauseData.objects) {
  preCacheObjects(pauseData.objects);
}
```
There's no point to make a new array _and_ call a method when there's no data to cache.

Reply:
True, that would also work.

Reply:
Reverted this and used `if` statements instead